setup_wordpress.sh

```diff
@@ -1,3 +1,91 @@
 #!/bin/bash
 
 echo 'This script should install and setup Wordpress'
+
+
+# Variables needed for configurations these can be edited to anything you want
+Dbname="wordpress"
+Dbuser="wpuser"
+Dbpass="Devops101"
+Wppass="!2three456."
+
+###############
+# nginx, mysql, WordPress.
+# WordPress should have already configured with theme twentynineteen and with created user admin with password !2three456..
+#####################
+
+# install packages
+sudo apt update
+sudo apt-get upgrade -y
+sudo apt install nginx -y
+sudo apt install mysql-server -y 
+sudo apt install php7.4 php7.4-fpm php7.4-mysql php7.4-curl php7.4-gd php7.4-intl php7.4-mbstring php7.4-soap php7.4-xml php7.4-xmlrpc php7.4-zip -y
+
+
+#configure firewall nginx
+sudo ufw enable
+sudo ufw allow 'Nginx HTTP'
+
+# install and configure mysql-server
+sudo mysql -e "CREATE DATABASE $Dbname DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;"
+sudo mysql -e "CREATE USER '$Dbuser'@'localhost' IDENTIFIED BY '$Dbpass';"
+sudo mysql -e "GRANT ALL ON $Dbname.* TO '$Dbuser'@'localhost';"
+sudo mysql -e "FLUSH PRIVILEGES;"
+
+# Permissions configurations for wordpress
+cd /tmp
+sudo wget https://wordpress.org/latest.tar.gz
+tar xzvf latest.tar.gz
+sudo cp /tmp/wordpress/wp-config-sample.php /tmp/wordpress/wp-config.php
+sudo mkdir -p /home/www/wordpress/
+sudo cp -a /tmp/wordpress/. /home/www/wordpress/
+sudo chown -R www-data:www-data /home/www/wordpress/
+sudo find /home/www/wordpress/ -type f -exec chmod 775 {} \;
+sudo find /home/www/wordpress/ -type f -exec chmod 644 {} \;
+
+# Create wordpress Virtual Host
+cat << EOF > /etc/nginx/sites-available/wordpress
+server {
+	listen 80;
+	server_name 10.0.2.15;
```

Comment:
what does this mean?

Reply:
hmm good question.
I was testing it constantly in my virtual box and probably forgot to change it.
Another reason why using a variable there would also be beneficial.

Reply:
BTW, just curious. Have you changed anything in the Vagrantfile?

Reply:
No i did not touch it, I only tested to see if the script worked by using the command vagrant up and sometimes vagrant ssh to see if the files were correctly installed. I am actually a bit confused now that I see that the server name is different then the actual ip address in the vagrantfile. Nginx and its virtual hosts is another level that I need to go deeper in to understand.